pairtools/pairtools_parse.py

```diff
@@ -822,18 +838,20 @@ def streaming_classify(instream, outstream, chromosomes, min_mapq, max_molecule_
                                   algn1['type'] + algn2['type'])
 
             if out_alignments_stream:
-                write_all_algnments(prev_read_id, all_algns1, all_algns2, out_alignments_stream)
+                write_all_algnments(prev_readID, all_algns1, all_algns2, out_alignments_stream)
             
             sams1.clear()
             sams2.clear()
 
         if line is not None:
```

Comment:
if I do `--readid-transform 1` - we'd keep "pushing" sam-s as if there was only one read ...
how would it break ? memory overuse ?

should we put a limit on the number of reads/alignments per `sam1/sam2` ? with a simple counter ?

or maybe do check on the `readid-transform` function ?